gcp/compute/resources.py

```diff
@@ -14,34 +14,43 @@ def instances():
 
 
 def clusters():
-    parent = "projects/" + gcp_client.get_project_id() + "/locations/-"
-    return gcp_client.list(
-        "container",
-        "projects.locations.clusters",
-        results_key="clusters",
-        call_kwargs={"parent": parent},
-    )
+    results = []
+    for project_id in gcp_client.project_list:
+        results += gcp_client.list(
+            "container",
+            "projects.locations.clusters",
+            results_key="clusters",
+            call_kwargs={"parent": "projects/{}/locations/-".format(project_id)},
+        )
+    return results
 
 
 def networks_with_instances():
+    allInstances = instances()
+    inUseNetworks = []
```

Comment:
nit: snake_case would be more pythonic but if the API var names are camelCase that might be clearer